devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java

```diff
@@ -13,4 +50,156 @@ public static String getDerivedPath(String className) {
                 className.substring(className.lastIndexOf(".") + 1));
         return "/" + resourceClassName[0].toLowerCase();
     }
+
+    public static void setupQuarkusJsonPlatformDescriptor(
+            RepositorySystem repoSystem, RepositorySystemSession repoSession, List<RemoteRepository> repos,
+            String platformGroupId, String platformArtifactId, String defaultPlatformVersion,
+            final Log log)
```

Comment:
Why log is final but not the rest 😅

Reply:
It doesn't have to be now. Originally, it was referenced from an anonymous class, so it had to be final.